aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -755,6 +755,7 @@ _scaled_mm_out_cuda(const Tensor& mat1, const Tensor& mat2,
   TORCH_CHECK(mat1.scalar_type() != ScalarType::Float8_e5m2 || mat2.scalar_type() != ScalarType::Float8_e5m2,
         "Multiplication of two Float8_e5m2 matrices is not supported");
   if (bias) {
+    TORCH_CHECK(out.scalar_type() != kFloat, "Bias is not supported when out_dtype is set to Float32");
```

Comment:
should this check that `mat1`/`mat2` are `float8`?

Reply:
yeah, good catch

Reply:
actually this is not needed because of https://github.com/pytorch/pytorch/blob/424f8df8add13eb8698b683cfece9e22c38866b2/aten/src/ATen/native/cuda/Blas.cpp#L752-L753